src/plugins/fakeNitro.tsx

```diff
@@ -483,7 +685,7 @@ export default definePlugin({
                     }
 
                     delete extra.stickerIds;
-                    messageObj.content += " " + link;
+                    messageObj.content += " " + link + `&name=${sticker.name}`;
```

Comment:
Not sure if names can contain non urlsafe characters but better be safe than sorry

```suggestion
                    messageObj.content += " " + link + `&name=${encodeURIComponent(sticker.name)}`;
```